tests/Issues/NamespacedUse/Fixture/namespaced_constant.php.inc

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Apple {
+    const Banana = 'test';
+}
+
+namespace {
```

Comment:
we don't support 2+ namespace in single file for add use statement, while add + remove can be enabled at the same time, move source file to new file under `Source` instead, eg: `tests/Issues/NamespacedUse/Source`, add namespace: `Rector\Tests\Issues\NamespacedUse\Source`

Reply:
I split the test. Please let me know if you find any other test cases that would be valuable to add.